torchinductor/ir.py

```diff
@@ -1497,6 +1498,106 @@ def _apply_loop_reordering(index_vars, sizes, memory_addrs):
         sizes = [sizes[i] for i in order]
         return sizes, inverse_reorder(order)
 
+    @staticmethod
+    def _apply_loop_reordering_channel_last(index_vars, sizes):
```

Comment:
If the tensor is already channels last order will this do the incorrect thing?

Reply:
https://github.com/pytorch/torchdynamo/pull/461/files#diff-40b1a2d97e5bb20858ee792e1cb03d817e7264d69982d0127a403740f5378903R227-R235, it will check if the node is channel last format, if not, then it will call `reorder_channel_last`, and `reorder_channel_last` will call `_apply_loop_reordering_channel_last`. Do you think it is safe or do we need to add assertion inside the function?